torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -34,6 +34,19 @@ TensorView* maybe_broadcast_inner_to_rank(TensorView* t, size_t rank) {
   return t;
 }
 
+TensorView* maybe_broadcast_outter_to_rank(TensorView* t, size_t rank) {
```

Comment:
Can't we just use `std::vector<Val*> maybeBroadcast(const std::vector<Val*>& vals)` instead?